app/server.js

```diff
@@ -27,6 +28,17 @@ export async function createHttpServer() {
   app.use(compression())
   app.use(bodyParser.json())
 
+  app.use(
+    promBundle({
+      includePath: true,
+      promClient: {
+        collectDefaultMetrics: {
+          prefix: 'identity_service_',
```

Comment:
```suggestion
          prefix: 'dscp_identity_service_',
```

Reply:
@dblane-digicatapult `dscp-ipfs` just uses `ipfs_` is that an issue?